apps/condo/domains/resident/schema/SuggestServiceProviderService.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Generated by `createservice organization.SuggestServiceProviderByTinService --type queries`
+ */
+
+const { pick, uniqBy } = require('lodash')
+
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { MeterResourceOwner } = require('@condo/domains/meter/utils/serverSchema')
+const { SERVICE_PROVIDER_TYPE } = require('@condo/domains/organization/constants/common')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const access = require('@condo/domains/resident/access/SuggestServiceProviderService')
+const { MAX_RESIDENT_SUGGEST_SERVICE_PROVIDER_BY_WINDOW_SEC, RESIDENT_SUGGEST_SERVICE_PROVIDER_WINDOW_SEC } = require('@condo/domains/resident/constants')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+const checkLimits = async (uniqueField) => {
+    await redisGuard.checkCustomLimitCounters(
+        `suggest-service-provider-by-tin-${uniqueField}`,
```

Comment:
so the correct redis keys should look something like this:
`guard_counter:user:${id}:suggest_service_provider`